bin/simAstrom.py

```diff
@@ -21,6 +21,6 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 import lsst.pex.logging as pexLog
-from lsst.meas.simastrom.simAstrom3 import SimAstromTask
+from lsst.jointcal.jointcal3 import jointcalTask
```

Comment:
I didn't touch these since I don't think they're relevant for anything. Maybe just delete them?